components/supabase/sources/new-row-added/new-row-added.mjs

```diff
@@ -0,0 +1,110 @@
+import base from "../common/base.mjs";
+import {
+  DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+  ConfigurationError,
+} from "@pipedream/platform";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "supabase-new-row-added",
+  name: "New Row Added",
+  description: "Emit new event for every new row added in a table. [See documentation here](https://supabase.com/docs/reference/javascript/select)",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    ...base.props,
+    column: {
+      propDefinition: [
+        base.props.supabase,
+        "column",
+      ],
+      optional: true,
+    },
+    filter: {
+      propDefinition: [
+        base.props.supabase,
+        "filter",
+      ],
+      optional: true,
+    },
+    value: {
+      propDefinition: [
+        base.props.supabase,
+        "value",
+      ],
+      optional: true,
+    },
+    orderBy: {
+      propDefinition: [
+        base.props.supabase,
+        "column",
+      ],
+      label: "Order By",
+      description: "Column name to order by",
+    },
+    sortOrder: {
+      propDefinition: [
+        base.props.supabase,
+        "sortOrder",
+      ],
+    },
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+  },
+  methods: {
+    _getOffset() {
+      return this.db.get("offset") || 0;
+    },
+    _setOffset(offset) {
+      this.db.set("offset", offset);
+    },
+  },
+  async run() {
+    const {
```

Comment:
```suggestion
      description: "The column name to use as the row identifier.",
```

Reply:
Added a period at the end of the description to follow Google's Developer docs style guide.